src/application/redux/reducers/wallet-reducer.ts

```diff
@@ -176,6 +220,22 @@ export function walletReducer(
       };
     }
 
+    case ACTION_TYPES.LOCK_UTXO: {
+      const utxo = payload.utxo as UnblindedOutput;
+      return {
+        ...state,
+        lockedUtxos: {
+          ...state.lockedUtxos,
+          [toStringOutpoint(utxo)]: Date.now(),
+        },
+      };
+    }
+
+    case ACTION_TYPES.ADD_UNCONFIRMED_UTXOS: {
+      const { unconfirmedUtxos, accountID, network } = payload;
+      return addUnconfirmed(state)(unconfirmedUtxos, accountID, network);
```

Comment:
I think we should handle the case where the "unconfirmed utxo" is never confirmed (= the updater does not return the "real" coin). If the previous tx is not confirmed for any reasons, our "fake" coin could cause missing-inputs error later. 

As a solution, we could remove the unconf coin if not confirmed after 1 min for instance?

Reply:
A broacasted TX can indeed being "replaced" by new txid indeed (replace by fee), so the change output can be different, so yes current virtual output can change.

Maybe we can move this it in a specific ticket and address it later?